examples/web/webui.py

```
@@ -6,150 +6,288 @@
-    parser.add_argument('--server_name', type=str, default='0.0.0.0', help='Server name')
-    parser.add_argument('--server_port', type=int, default=8080, help='Server port')
-    parser.add_argument('--root_path', type=str, default=None, help='Root Path')
-    parser.add_argument('--custom_path', type=str, default=None, help='the custom model path')
-    args = parser.parse_args()
 
-    print("loading ChatTTS model...")
-    global chat
-    chat = ChatTTS.Chat()
+    parser = argparse.ArgumentParser(description="ChatTTS demo Launch")
+    parser.add_argument(
+        "--server_name", type=str, default="0.0.0.0", help="server name"
```

Ruff (0.15.9)

[error] Possible binding to all interfaces

(S104)